src/sentry/api/base.py

```diff
@@ -160,6 +161,25 @@ def load_json_body(self, request):
         except json.JSONDecodeError:
             return
 
+    def _create_api_access_log(self):
+        """
+        Create a log entry to be used for api metrics gathering
+        """
+        try:
+            token_type = str(getattr(self.request.auth, "__class__", None))
+            log_metrics = dict(
+                method=self.request.method,
+                view=str(self.request.parser_context["view"].__class__),
+                response=self.response.status_code,
+                user=str(self.request.user),
```

Comment:
What will the `user` be if something like an integration or sentry app calls the API? Also, is there anyway to identify where the request came from if the user is `AnonymousUser` like how you showed in your example? 

Just making sure we can use this to identify when users hammer our API.

Reply:
For the first question, I don't know what the user will be, but we will after we start logging it :)

There might be a way to track the IP address of the caller, but generally if we're getting an AnonymousUser call then that means it's an unauthenticated user. The objective here is to add more detail in subsequent iterations.